zanata-war/src/main/java/org/zanata/rest/editor/service/UserService.java

```diff
@@ -69,7 +74,7 @@ public Response getUserInfo(String username) {
     }
 
     @Override
-    public User generateUser(String username) {
+    public User generateUser(String username, boolean includeEmail) {
```

Comment:
Why not just leave the method unchanged and use applicationConfiguration.isDisplayUserEmail() to get the boolean value again.